examples/c/ocp_qp.c

```diff
@@ -128,16 +127,16 @@ int main() {
      ************************************************/
 
     double res[4];
-    ocp_qp_inf_norm_residuals(&dims, qp_in, qp_out, res);
+    ocp_qp_inf_norm_residuals(dims->orig_dims, qp_in, qp_out, res);
```

Comment:
The struct members should not be accessed directly, as this will break in case of changes.
The correct solution is the one from the other comment, with two dims structures, one for the QP (to use here) and another one for the solver.